replicated-security/banksy-testnet-3/README.md

```diff
@@ -0,0 +1,97 @@
+* Genesis file (with CCV): `Available soon`
+* Pre-transition Composable binary: 
+   * Version: [v5.0.0](https://github.com/notional-labs/Composable-ICS-tesnet/raw/main/binaries/v5.0.0/centaurid)
+   * SHA256: `ef92568fed75492cee9b634ba368ed17e7c82737433e6918bc96f20e51a2b089`
+* Composable’s GitHub repository: https://github.com/notional-labs/composable-centauri
+* Go version: `1.19`
+* Persistent Peers = `c0f197bdf6c4a4a16eb9db112d1ec9545336fd43@168.119.91.22:2250`
+* Chain ID: `banksy-testnet-3`
+* Post-upgrade Composable binary commit (run with this binary after the upgrade):
+   * Version: [v6.0.0-ics](https://github.com/notional-labs/composable-centauri/releases/download/v6.0.0-ics/centaurid)
+   * SHA256: `43045ea6208cf11409ce1a2e37fc7a9723f7199f4c34c3c371a39ed3bdce376b`
+- Public Notional endpoints: 
+    - RPC: `https://rpc-banksy.notional.ventures:443`
+    - API: `https://api-banksy.notional.ventures:443`
+    - gRPC: `https://grpc-banksy.notional.ventures:443`
+- Block Explorer: `https://explorer.nodexcapital.com/banksy-testnet`
+
+### IBC data
+Available soon
+
+### Joining berfore transition
+All provider validators must join `banksy-testnet-3` before the upgrade and `spawn_time`, using released binary:
+
+```bash
+export PATH=$PATH:$HOME/go/bin
+wget https://github.com/notional-labs/composable-centauri/releases/download/v5.0.0/centaurid -O $HOME/go/bin.centaurid
+chmod +x $HOME/go/bin.centaurid
+sudo wget -P /usr/lib https://github.com/CosmWasm/wasmvm/raw/main/internal/api/libwasmvm.x86_64.so
+centaurid version # v5.0.0
+centaurid init <moniker> --chain-id banksy-testnet-3
+
+# Genesis file without ccv module
+wget https://raw.githubusercontent.com/notional-labs/Composable-ICS-tesnet/main/genesis.json -O $HOME/.banksy/config/genesis.json
+
+# run node
+centaurid start --p2p.seeds c0f197bdf6c4a4a16eb9db112d1ec9545336fd43@168.119.91.22:2250
+```
+Or building binary from source:
+```bash
+export PATH=$PATH:$HOME/go/bin
+cd $HOME
+git clone https://github.com/notional-labs/composable-centauri
+cd composable-centauri
+git checkout v5.0.0 # Using v5.0.0
+make install
+centaurid version # v5.0.0
+```
+
+### Upgrade and apply new genesis file
+The node will start running until the upgrade height, at which the node will halt. Detail about timeline and instruction can be seen at **Launch Stages** section. To download new genesis file at `spawn_time`, and apply new version after upgrade height, see following commands:
+```bash
+# Download new genesis-ccv at `spawn_time`
+wget -O $HOME/.banksy/config/genesis-ccv.json https://raw.githubusercontent.com/cosmos/testnets/master/replicated-security/banksy-testnet-3/genesis-ccv.json
+```
+
+```bash
+# Installing centaurid v6.0.0-ics
+export PATH=$PATH:$HOME/go/bin
+cd $HOME
+git clone https://github.com/notional-labs/composable-centauri
+cd composable-centauri
+git checkout v6.0.0-ics # Using v6.0.0-ics
+make install
+centaurid version # v6.0.0-ics
```

Comment:
This returns `v5.1.0`, can you double-check please?

Reply:
yes it is a problem in our code. Our team are fixing that